02/app.js

```diff
@@ -1,2 +1,22 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+for(let i = 0; i <= x; i++) {
+    result += i;
+    console.log(result);
+}
+
+
+
+// let x = 0;
+// let result = 0;
+// while (x < 5) {
+//   x++;
+//   result += x;
+//   console.log(result)
+// }
```

Comment:
Zobacz, że pętla `while` nie posiada dodatkowej zmiennej tzw. sterującej.
Musisz ją sobie stworzyć (np. `let j = 0 `) i to ją porównywać np. `while (j < x)` i inkrementować wew. tj. `j++`